### ISSUE TITLE
Android splash screen image is cutted

### ISSUE THREAD

**Issue by felix4321** [zoontek/react-native-bootsplash#290]
### Bug summary

I want to use a splash image with 240 width. On iOS it works perfectly (see first screenshot). On Android the edges are cutted (see second screenshot). 

I generated the files with: yarn react-native generate-bootsplash --logo-width 240

If i set the image width to 100 everything works perfectly.

Is it a problem with the bootsplash lib or the android-api (which seems to have some limitations)?

iOS:
![IMG_0632](https://user-images.githubusercontent.com/50043030/144409379-f79ec788-01ce-4322-8a9c-45071cb954e1.PNG)

Android (tested on multiple devices):
![Screenshot_20211202-113842](https://user-images.githubusercontent.com/50043030/144409389-4ed96ea8-5aeb-4b1b-a1e0-823cb029d4a5.jpg)

### Library version

4.0.2

### Environment info

```shell
info Fetching system and libraries information...
System:
    OS: macOS 12.0.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 27.73 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.0.1 - /var/folders/9z/j13cff7j5j18r5nxwg_w70wc0000gn/T/yarn--1638441772806-0.8722354824138727/node
    Yarn: 1.22.17 - /var/folders/9z/j13cff7j5j18r5nxwg_w70wc0000gn/T/yarn--1638441772806-0.8722354824138727/yarn
    npm: 8.1.0 - /usr/local/bin/npm
    Watchman: 2021.11.01.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.0.1, iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0
    Android SDK:
      API Levels: 27, 29, 30, 31
      Build Tools: 28.0.0, 28.0.3, 29.0.2, 29.0.3, 30.0.2, 31.0.0, 32.0.0
      System Images: android-31 | Google APIs Intel x86 Atom_64
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.2 AI-202.7660.26.42.7486908
    Xcode: 13.1/13A1030d - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.13 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.66.3 => 0.66.3 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
✨  Done in 4.12s.
```

### Steps to reproduce

1. yarn react-native generate-bootsplash --logo-width 240

**Comment by zoontek** [comment]
You can't, there's a mask: https://developer.android.com/guide/topics/ui/splash-screen

During the asset generation, a warning is even given: https://github.com/zoontek/react-native-bootsplash/blob/master/react-native.config.js#L49

**Comment by sadikyalcin** [comment]
I have the same issue. Initially the logo is cut off and then a second or so after, it's corrected. I've set the width to 190 during the asset generation.

I've managed to get screenshots;

![Screenshot_20211213-120634](https://user-images.githubusercontent.com/8109619/145809473-14557e31-d5a2-4697-a8db-46635ef1d7ab.jpg)

![Screenshot_20211213-120626](https://user-images.githubusercontent.com/8109619/145809507-b39b407a-706e-49a6-b6fe-4d8bcd1346eb.jpg)
